smtpsrv/server.go

```diff
@@ -2,12 +2,37 @@ package smtpsrv
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"net"
+	"strings"
 	"time"
 
 	"github.com/emersion/go-smtp"
+	"github.com/target/goalert/util/log"
 )
 
+// SMTPLogger implements the smtp.Logger interface using the main app Logger
+type SMTPLogger struct {
+	logger *log.Logger
+}
+
+// Printf adheres to smtp.Server's Logger interface while filtering out ECONNRESET errors caused by TCP health checks
+func (l *SMTPLogger) Printf(format string, v ...interface{}) {
```

Comment:
Let's add a TODO for these
```suggestion
// Printf adheres to smtp.Server's Logger interface.
//
// TODO: Uses string compare to filter out errors caused by TCP health checks,
// remove once https://github.com/emersion/go-smtp/issues/236 has been fixed.
func (l *SMTPLogger) Printf(format string, v ...interface{}) {
```

Reply:
Added TODO at conditional under each func